### ISSUE TITLE
Add support for version 10.0.18252.1000

### ISSUE THREAD

**Issue by viwar0** [stascorp/rdpwrap#589]
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/2483097/termsrv.zip)

**Comment by JEmlay** [comment]
Agreed, needs to be updated for 1809+.  Hopefully it's at least being worked on.

**Comment by liangshiw** [comment]
Agreed, needs to be updated for 1809+. Hopefully it's at least being worked on.

**Comment by sebaxakerhtc** [comment]
> 
> 
> [termsrv.zip](https://github.com/stascorp/rdpwrap/files/2483097/termsrv.zip)

https://github.com/sebaxakerhtc/rdpwrap.ini/commit/3025f408e4fe0935ecd28d11433e155c8dc54308
